v20.2/authorization.md

```diff
@@ -97,6 +97,16 @@ A user or role that is a member of the role by association.
 
 Example: A is a member of C ... is a member of B where "..." is an arbitrary number of memberships.
 
+## Object ownership
+
+<span class="version-tag">New in v20.2</span> All CockroachDB objects must have owners. The user that created the object is the default owner of the object and has `ALL` privileges on the object. Similarly, any roles that are members of the owner role also have all privileges on the object.
```

Comment:
I'm not really familiar with what "objects" means in terms of a SQL database.  Do we mean things such as databases, tables, etc.?

If so I think it would be good to add something like

> All CockroachDB objects (such as databases, tables ...) ...